packages/server/src/github/api.js

```diff
@@ -0,0 +1,231 @@
+const assert = require('assert');
+const _ = require('lodash');
+const request = require('request');
+const crypto = require('crypto');
+
+const config = require('../config');
+
+class API {
+  /**
+   * @public
+   */
+  constructor() {
+    this.repoOwner = config.GITHUB_REPO_OWNER;
+    this.repoName = config.GITHUB_REPO_NAME;
+    this.token = config.GITHUB_PERSONAL_ACCESS_TOKEN;
+    this.webhookSecret = config.GITHUB_WEBHOOK_SECRET;
+    this.apiURL = `https://api.github.com`;
```

Comment:
I feel like this should go into a constants file somewhere, at the very least be a constant on the top of the file. I don't like magic numbers.

Reply:
Do you mean the last string with GitHub URL?

Reply:
yep

Reply:
Fixed.